components/Footer.js

```diff
@@ -1,13 +1,11 @@
 import Image from 'next/image';
 import styled from 'styled-components';
-import backwardicon from '../Assets/Icons/backward.ico';
-import forwardicon from '../Assets/Icons/forward.ico';
 
 export default function Footer() {
   return (
     <>
       <Footerbody>
-        <Backward
+        {/* <Backward
```

Comment:
you can remove the comment and the parts within

Reply:
Done!